File: client/iih_stub/iih.ci_stub/iih.ci.rcm/contagionmessage/i/IContagionMessageService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using iih.ci.rcm.contagion.d;

namespace iih.ci.iih.ci.rcm.contagionmessage.i
{
    public interface IContagionMessageService
    {
        string SendMessage(ContagionDO condo);
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/rcm/contagionmessage/i/IContagionMessageService.java
package iih.ci.rcm.contagionmessage.i;

import iih.ci.rcm.contagion.d.ContagionDO;
import xap.mw.core.data.BizException;

public interface IContagionMessageService {
	public int sentMessage(ContagionDO condo) throws BizException;

}
